### PR TITLE
Ser 219 maize recommendation

### PR DESCRIPTION
**Pull request by Bahatiroben**
this is the first part of the maize recommendation all it does is it triggers the 2 way messages services. but since the service is not live yet, it can't be triggered. so it will log the error on slack that it can't trigger a non-live service

test from here
https://telerivet.com/p/0c6396c9/service/SV7684c662f118f82e/edit
account number: 26215997
maize recommendation is option 11 on screen 2.

### CODE REVIEW

**Review by shejasabin — APPROVED** [review]

File: ke-legacy/utils/menus/populate-menu/mainMenu.js [hunk 1]
```diff
@@ -89,5 +89,12 @@ module.exports = [
         'option_name': 'view_group_repayment',
         'end_date': project.vars.end_view_group_repayment,
         'start_date': project.vars.start_view_group_repayment
+    },
+    {
+        'en-ke': 'Maize Recommendations',
+        'sw': 'Maize Recommendations',
```
Comment (nyabongo) [inline ke-legacy/utils/menus/populate-menu/mainMenu.js:95]: sw is not translated here?

File: maize-recommendation/translations/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    'message_notification': {
+        'en': 'A SMS has been sent to your phone. Messages are free! But please delete unwanted SMS in your phone to make space for incoming.',
+        'en-ke': 'A SMS has been sent to your phone. Messages are free! But please delete unwanted SMS in your phone to make space for incoming.',
+        'sw': 'A SMS has been sent to your phone. Messages are free! But please delete unwanted SMS in your phone to make space for incoming.'
```
Comment (nyabongo) [inline maize-recommendation/translations/index.js:5]: No Translation here either?

File: ke-legacy/utils/menus/populate-menu/mainMenu.js (hunk 1 above)
Comment (Bahatiroben) [reply ke-legacy/utils/menus/populate-menu/mainMenu.js:95]: this is the first part of the PR. so, as the translations are not available yet, I thought I could put it up with english, then Once the translations are available, I will replace them on the last PR.

File: maize-recommendation/translations/index.js (hunk 2 above)
Comment (Bahatiroben) [reply maize-recommendation/translations/index.js:5]:
```suggestion
        'sw': 'A SMS has been sent to your phone. Messages are free! But please delete unwanted SMS in your phone to make space for incoming.'this is the first part of the PR. so, as the translations are not available yet, I thought I could put it up with english, then Once the translations are available, I will replace them on the last PR.
```

Comment (Bahatiroben) [reply maize-recommendation/translations/index.js:5]: The same for here.

File: ke-legacy/utils/menus/populate-menu/mainMenu.js (hunk 1 above)
Comment (nyabongo) [reply ke-legacy/utils/menus/populate-menu/mainMenu.js:95]: This is not a good pattern, an incorrect translation is a bug, we shouldn't be merging these. If a translation is missing make sure to bring it up as a blocker in the standups

**Review by nyabongo — APPROVED** [review]

**Pull request merged**